integration/fixtures/registries.conf

```diff
@@ -0,0 +1,26 @@
+[[registry]]
+url = "mirror.local"
+mirror = [
+    { url = "mirror-0.local" },
+    { url = "mirror-1.local" },
+    { url = "gcr.io/google-containers" },
+]
+
+[[registry]]
+url = "invalid.local"
+mirror = [
+    { url = "invliad-mirror-0.local" },
+    { url = "invliad-mirror-1.local" },
+]
+
+[[registry]]
+url = "gcr.local"
+prefix = "gcr.local/foo/bar"
+mirror = [
+    { url = "wrong-mirror-0.local" },
+    { url = "gcr.io/google-containers" },
+]
+
+[[registry]]
+url = "gcr.io"
+prefix = "gcr.io/google-containers"
```

Comment:
Is this used at all? Does this maybe exist to ensure that the result of `mirror.local/busybox` is not rewritten twice? If so, using a clearly-invalid `url` might be useful, and there definitely should be a comment.

Reply:
I adapted it as suggested.